mnefun/_mnefun.py

```diff
@@ -2330,3 +2332,144 @@ def plot_reconstruction(evoked, origin=(0., 0., 0.04)):
     axs[2, 0].set_ylabel('Reconstruction')
     fig.tight_layout()
     return fig
+
+
+def plot_chpi_snr_raw(raw, win_length, n_harmonics):
+    """Compute and plot cHPI SNR from raw data
+
+    Parameters
+    ----------
+    win_length : float
+        Length of window to use for SNR estimates (seconds). A longer window
+        will naturally include more low frequency power, resulting in lower
+        SNR.
+    n_harmonics : int
```

Comment:
Yep, for example. I could look at some 5 kHz data we just recorded. IIRC the line freq harmonics don't typically go very high, even though that may vary by country / electricity provider...